frontend/src/app/components/wp-inline-create/wp-inline-create.component.ts

```diff
@@ -56,15 +54,10 @@ import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/iso
 import {componentDestroyed, untilComponentDestroyed} from 'ng2-rx-componentdestroyed';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {FocusHelperService} from 'core-app/modules/common/focus/focus-helper';
-import {IWorkPackageCreateServiceToken} from "core-components/wp-new/wp-create.service.interface";
-import {CurrentUserService} from "core-components/user/current-user.service";
 import {WorkPackageInlineCreateService} from "core-components/wp-inline-create/wp-inline-create.service";
 import {Subscription} from 'rxjs';
 import {WorkPackageViewColumnsService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-columns.service";
-import {
-  WorkPackageEvent,
-  WorkPackageEventsService
-} from "core-app/modules/work_packages/events/work-package-events.service";
+import {WorkPackageChangeset} from "core-components/wp-edit/work-package-changeset";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/b6e4680ef80bc97abdec363477c560842ceb1115/frontend/src/app/components/wp-inline-create/wp-inline-create.component.ts#L61"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>